ARM binary release

<!-- Describe the feature you'd like. -->
I have a RPi 3B+ module，and i hope code-server will be running within this platform, thank u

> BTW one interesting but maybe not super-fun workaround is to build code-server but not package it into a binary and just use the loose files with an ARM version of Node:
> 
> ```
> node /path/to/build/out/vs/server/main.js
> ```

Followed this on my RPi 3b+ and received this error
`internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '../../bootstrap-amd'`
If you know how to fix it please reply :)

--- a/.drone.yml
+++ b/.drone.yml
@@ -11,17 +11,17 @@ steps:
    image: node:12
    commands:
      - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: build-alpine
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
    image: debian
@@ -50,17 +50,17 @@ steps:
    image: node:12
    commands:
      - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: build-alpine
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
    image: debian
@@ -90,16 +90,16 @@ steps:
    commands:
      - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
      - yarn
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
 
  - name: build-alpine
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
    image: debian

--- a/.drone.yml
+++ b/.drone.yml
@@ -24,15 +24,11 @@ steps:
      - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: debian
-   commands:
-     - apt-get update && apt-get install -y wget
-     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
-     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
-     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
-   environment:
-    GITHUB_TOKEN:
-      from_secret: gh-token
+   image: plugins/github-release
+   settings:
+    api_key:
+      from_secret: github_token
+    files: release/*.tar.gz
    when:
     event: tag
 
@@ -63,15 +59,11 @@ steps:
      - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: debian
-   commands:
-     - apt-get update && apt-get install -y wget
-     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
-     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
-     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
-   environment:
-    GITHUB_TOKEN:
-      from_secret: gh-token
+   image: plugins/github-release
+   settings:
+    api_key:
+      from_secret: github_token
+    files: release/*.tar.gz
    when:
     event: tag
 
@@ -102,14 +94,10 @@ steps:
      - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: debian
-   commands:
-     - apt-get update && apt-get install -y wget
-     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
-     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
-     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
-   environment:
-    GITHUB_TOKEN:
-      from_secret: gh-token
+   image: plugins/github-release
+   settings:
+    api_key:
+      from_secret: github_token
+    files: release/*.tar.gz
    when:
     event: tag

--- a/.drone.yml
+++ b/.drone.yml
@@ -32,6 +32,22 @@ steps:
    when:
     event: tag
 
+- name: docker-publish
+  image: banzaicloud/drone-kaniko
+  settings:
+    username:
+      from_secret: docker_user
+    password:
+      from_secret: docker_password
+    tags:
+      - latest
+      - $DRONE_TAG
+    build_args:
+      - codeServerVersion=$DRONE_TAG
+      - vscodeVersion=1.41.1
+  when:
+    event: tag
+
 ---
 kind: pipeline
 type: docker
@@ -65,6 +81,22 @@ steps:
       from_secret: github_token
     files: release/*.tar.gz
    when:
+    event: tag
+
+- name: docker-publish
+  image: banzaicloud/drone-kaniko
+  settings:
+    username:
+      from_secret: docker_user
+    password:
+      from_secret: docker_password
+    tags:
+      - latest
+      - $DRONE_TAG
+    build_args:
+      - codeServerVersion=$DRONE_TAG
+      - vscodeVersion=1.41.1
+  when:
     event: tag
 
 # ---

--- a/.drone.yml
+++ b/.drone.yml
@@ -32,21 +32,21 @@ steps:
    when:
     event: tag
 
-- name: docker-publish
-  image: banzaicloud/drone-kaniko
-  settings:
-    username:
-      from_secret: docker_user
-    password:
-      from_secret: docker_password
-    tags:
-      - latest
-      - ${DRONE_TAG}
-    build_args:
-      - codeServerVersion=${DRONE_TAG}
-      - vscodeVersion=1.41.1
-  when:
-    event: tag
+ - name: docker-publish
+   image: banzaicloud/drone-kaniko
+   settings:
+     username:
+       from_secret: docker_user
+     password:
+       from_secret: docker_password
+     tags:
+       - latest
+       - ${DRONE_TAG}
+     build_args:
+       - codeServerVersion=${DRONE_TAG}
+       - vscodeVersion=1.41.1
+   when:
+     event: tag
 
 ---
 kind: pipeline
@@ -83,21 +83,21 @@ steps:
    when:
     event: tag
 
-- name: docker-publish
-  image: banzaicloud/drone-kaniko
-  settings:
-    username:
-      from_secret: docker_user
-    password:
-      from_secret: docker_password
-    tags:
-      - latest-arm64
-      - ${DRONE_TAG}-arm64
-    build_args:
-      - codeServerVersion=$DRONE_TAG
-      - vscodeVersion=1.41.1
-  when:
-    event: tag
+ - name: docker-publish
+   image: banzaicloud/drone-kaniko
+   settings:
+     username:
+       from_secret: docker_user
+     password:
+       from_secret: docker_password
+     tags:
+       - latest-arm64
+       - ${DRONE_TAG}-arm64
+     build_args:
+       - codeServerVersion=$DRONE_TAG
+       - vscodeVersion=1.41.1
+   when:
+     event: tag
 
 # ---
 # kind: pipeline

--- a/.drone.yml
+++ b/.drone.yml
@@ -67,37 +67,37 @@ steps:
    when:
     event: tag
 
----
-kind: pipeline
-type: docker
-name: build-arm
-
-platform:
-  arch: arm
-
-steps:
-
- - name: build
-   image: node:12
-   commands:
-     - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
-     - yarn
-     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
-
- - name: build-alpine
-   image: node:12-alpine
-   commands:
-     - apk add libxkbfile-dev libsecret-dev build-base git
-     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
-     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
-     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
-
- - name: publish
-   image: plugins/github-release
-   settings:
-    api_key:
-      from_secret: github_token
-    files: release/*.tar.gz
-   when:
-    event: tag
+# ---
+# kind: pipeline
+# type: docker
+# name: build-arm
+#
+# platform:
+#  arch: arm
+#
+# steps:
+#
+# - name: build
+#   image: node:12
+#   commands:
+#     - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
+#     - yarn
+#     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+#     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+#
+# - name: build-alpine
+#   image: node:12-alpine
+#   commands:
+#     - apk add libxkbfile-dev libsecret-dev build-base git
+#     - if [ -z $DRONE_TAG ]; then yarn build 1.41.1 daily ; else yarn build 1.41.1 $DRONE_TAG; fi
+#     - if [ -z $DRONE_TAG ]; then yarn binary 1.41.1 daily ; else yarn binary 1.41.1 $DRONE_TAG; fi
+#     - if [ -z $DRONE_TAG ]; then yarn package 1.41.1 daily ; else yarn package 1.41.1 $DRONE_TAG; fi
+#
+# - name: publish
+#   image: plugins/github-release
+#   settings:
+#    api_key:
+#      from_secret: github_token
+#    files: release/*.tar.gz
+#   when:
+#    event: tag

--- a/.drone.yml
+++ b/.drone.yml
@@ -1,0 +1,103 @@
+kind: pipeline
+type: docker
+name: build-amd64
+
+platform:
+  arch: amd64
+
+steps:
+
+ - name: build
+   image: node:12
+   commands:
+     - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: build-alpine
+   image: node:12-alpine
+   commands:
+     - apk add libxkbfile-dev libsecret-dev build-base git
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: publish
+   image: plugins/github-release
+   settings:
+     api_key:
+        from_secret: gh-token
+     files: '*.tar.gz'
+   when:
+     event: tag
+
+---
+kind: pipeline
+type: docker
+name: build-arm64
+
+platform:
+  arch: arm64
+
+steps:
+
+ - name: build
+   image: node:12
+   commands:
+     - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: build-alpine
+   image: node:12-alpine
+   commands:
+     - apk add libxkbfile-dev libsecret-dev build-base git
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: publish
+   image: plugins/github-release
+   settings:
+     api_key:
+        from_secret: gh-token
+     files: '*.tar.gz'
+   when:
+     event: tag
+
+---
+kind: pipeline
+type: docker
+name: build-arm
+
+platform:
+  arch: arm
+
+steps:
+
+ - name: build
+   image: node:12
+   commands:
+     - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: build-alpine
+   image: node:12-alpine
+   commands:
+     - apk add libxkbfile-dev libsecret-dev build-base git
+     - yarn
+     - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+
+ - name: publish
+   image: plugins/github-release
+   settings:
+     api_key:
+        from_secret: gh-token
+     files: '*.tar.gz'
+   when:
+     event: tag

--- a/.drone.yml
+++ b/.drone.yml
@@ -41,9 +41,9 @@ steps:
       from_secret: docker_password
     tags:
       - latest
-      - $DRONE_TAG
+      - ${DRONE_TAG}
     build_args:
-      - codeServerVersion=$DRONE_TAG
+      - codeServerVersion=${DRONE_TAG}
       - vscodeVersion=1.41.1
   when:
     event: tag
@@ -91,8 +91,8 @@ steps:
     password:
       from_secret: docker_password
     tags:
-      - latest
-      - $DRONE_TAG
+      - latest-arm64
+      - ${DRONE_TAG}-arm64
     build_args:
       - codeServerVersion=$DRONE_TAG
       - vscodeVersion=1.41.1

--- a/.drone.yml
+++ b/.drone.yml
@@ -11,26 +11,30 @@ steps:
    image: node:12
    commands:
      - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
-     - yarn
      - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
      - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: build-alpine
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - yarn
      - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
      - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: plugins/github-release
-   settings:
-     api_key:
-        from_secret: gh-token
-     files: '*.tar.gz'
+   image: debian
+   commands:
+     - apt-get update && apt-get install -y wget
+     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
+     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
+     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
+   environment:
+    GITHUB_TOKEN:
+      from_secret: gh-token
    when:
-     event: tag
+    event: tag
 
 ---
 kind: pipeline
@@ -46,26 +50,30 @@ steps:
    image: node:12
    commands:
      - apt update && apt install -y build-essential git libsecret-1-dev libx11-dev libxkbfile-dev
-     - yarn
      - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
      - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: build-alpine
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - yarn
      - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
      - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: plugins/github-release
-   settings:
-     api_key:
-        from_secret: gh-token
-     files: '*.tar.gz'
+   image: debian
+   commands:
+     - apt-get update && apt-get install -y wget
+     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
+     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
+     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
+   environment:
+    GITHUB_TOKEN:
+      from_secret: gh-token
    when:
-     event: tag
+    event: tag
 
 ---
 kind: pipeline
@@ -89,15 +97,19 @@ steps:
    image: node:12-alpine
    commands:
      - apk add libxkbfile-dev libsecret-dev build-base git
-     - yarn
      - if [ ! -n $DRONE_TAG ]; then yarn build 1.41.1 daily; else yarn build 1.41.1 $DRONE_TAG; fi
      - if [ ! -n $DRONE_TAG ]; then yarn binary 1.41.1 daily; else yarn binary 1.41.1 $DRONE_TAG; fi
+     - if [ ! -n $DRONE_TAG ]; then yarn package 1.41.1 daily; else yarn package 1.41.1 $DRONE_TAG; fi
 
  - name: publish
-   image: plugins/github-release
-   settings:
-     api_key:
-        from_secret: gh-token
-     files: '*.tar.gz'
+   image: debian
+   commands:
+     - apt-get update && apt-get install -y wget
+     - wget -qO -  https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz | tar -xvz
+     - chmod -R 777 ghr_v0.12.0_linux_amd64/*
+     - ./ghr_v0.12.0_linux_amd64/ghr "$DRONE_TAG" ./release/
+   environment:
+    GITHUB_TOKEN:
+      from_secret: gh-token
    when:
-     event: tag
+    event: tag

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,18 +15,6 @@ before_install:
 # Don't build on tags because we'll already have built the commit.
 jobs:
   include:
-    - name: "Linux build"
-      os: linux
-      dist: trusty
-      env: TARGET="linux" PUSH_DOCKER="true"
-      if: tag IS blank
-      script: scripts/ci.bash
-    - name: "Alpine build"
-      os: linux
-      dist: trusty
-      env: TARGET="alpine"
-      if: tag IS blank
-      script: scripts/ci.bash
     - name: "MacOS build"
       os: osx
       if: tag IS blank

--- a/Dockerfile
+++ b/Dockerfile
@@ -52,6 +52,11 @@ RUN mkdir -p /home/coder/project
 # To avoid EACCES issues on f.ex Crostini (ChromeOS)
 RUN mkdir -p /home/coder/.local/share/code-server
 
+# FIXME: permission fix required for bind-mounts
+# See GH-1270
+RUN sudo chmod -R g+rw /home/coder/project && sudo chmod -R g+rw /home/coder/.local/share/code-server && \
+    sudo chown -R coder:coder /home/coder
+
 WORKDIR /home/coder/project
 
 # This ensures we have a volume mounted even if the user forgot to do bind

